src/cmd/common/common.go

```diff
@@ -0,0 +1,85 @@
+package common
+
+import (
+	"flag"
+	"fmt"
+	"log/slog"
+	"os"
+	"registry-stable/internal/github"
+	"registry-stable/internal/module"
+	"registry-stable/internal/provider"
+)
+
+type CLI struct {
+	Logger          *slog.Logger
+	Github          github.Client
+	ModuleDataDir   string
+	ProviderDataDir string
+}
+
+func Parse() CLI {
+	moduleDataDir := flag.String("module-data", "../modules", "Directory containing the module data")
+	providerDataDir := flag.String("provider-data", "../providers", "Directory containing the provider data")
```

Comment:
Whilst these 2 flags are common between the 2 commands right now, they may not be common across everything we have planned in the future. I think this is a bit of an over-optimization that isn't yet needed. However I'm happy to discuss this more.

Reply:
I'd also say that it shall be a single flag `"--output"` and the cmd's are rather split on the cmd level:
```commandline
go run ./cmd/bumper/provider --output=/tmp/foo
go run ./cmd/bumper/module --output=/tmp/bar
```

alternatively:

```commandline
go run ./cmd/bumper provider --output=/tmp/foo
go run ./cmd/bumper module --output=/tmp/foo
```